borgmatic/commands/borgmatic.py

```diff
@@ -616,10 +617,27 @@ def collect_configuration_run_summary_logs(configs, arguments):
         if 'extract' in arguments or 'mount' in arguments:
             validate.guard_single_repository_selected(repository, configs)
 
-        validate.guard_configuration_contains_repository(repository, configs)
+        if 'bootstrap' not in arguments:
+            validate.guard_configuration_contains_repository(repository, configs)
     except ValueError as error:
         yield from log_error_records(str(error))
         return
+    
+    if 'bootstrap' in arguments:
+        # no configuration file is needed for bootstrap
+        local_borg_version = borg_version.local_borg_version({}, 'borg')
+        try:
+            borgmatic.actions.bootstrap.run_bootstrap(arguments['bootstrap'], arguments['global'], local_borg_version)
+            yield logging.makeLogRecord(
+                dict(
+                    levelno=logging.INFO,
+                    levelname='INFO',
+                    msg='Bootstrap successful',
+                )
+            )
+        except (CalledProcessError, ValueError, OSError) as error:
```

Comment:
I don't think this would catch your `JSONDecodeError` or a `KeyError` exception types. So one option is to catch them and re-raise `ValueError`s within `get_config_paths()` or `run_bootstrap()`.. or you can just catch the expected exception types here.